db/seeds.rb

```diff
@@ -0,0 +1,33 @@
+require "bundler/setup"
+
+require "pg"
+require "active_record"
+require "pry"
+
+require_relative "connection"
+require_relative "../models/apartment"
+require_relative"../models/tenant"
+
+Apartment.create address ([
+  {address: "123 Main Street", monthy_rent: 1900, sqft: 650 num_beds: 1, num_baths: 1},
+  {address: "123 Steve Street", monthy_rent: 2200, sqft: 950 num_beds: 2, num_baths: 1.5},
+  {address: "256 Jason Street" monthly_rent: 3500, sqft: 1200, numb_beds: 3, num_baths: 2.5}
+  ])
+
+  Apartment.tenant.create ([
+    {name: "Jimmy", age: 23, gender: "Male"},
+    {name: "George", age: 22,  gender: "Male"},
+    {name: "Jaclyn", age: 25, gender: "Female"},
+    {name: "Tanya", age: 28, gender: "Female"},
+    {name: "Tommy", age: 45, gender: "Male"}
+    ])
```

Comment:
You will never call `.tenants` (note I made it plural here!) on the `Apartment` class.

Association helper methods (like `.tenants`) should only be called on an instance variable (e.g., @apartment, which contains one instance of an apartment).